packages/types/src/schemas/part.ts

```diff
@@ -3,7 +3,12 @@ import { baseSchema } from './common.js';
 
 export const partSchema = baseSchema.extend({
   type: z.literal('part'),
-  chapters: z.array(z.string()).optional(),
+  chapters: z
+    .array(z.string())
+    .optional()
+    .describe(
+      'The list of chapters in this part. The order in this array defines the order of the chapters. If not specified a folder-based numbering system is used instead.',
```

Comment:
```suggestion
      'The list of chapters in this part. The order of this array defines the order of the chapters. If not specified a folder-based numbering system is used instead.',
```

Reply:
I'll search whole codebase for `The order in this array` as this sentence is widely used.